README.md

```diff
@@ -1 +1,201 @@
-# Blockchain Database Server.
+# This is proposed design of Blockchain based Database
+
+![High level architecture](figures/Architecture.gif)
+
+## Assumptions:
+
+- During PoC we expose a KV store APIs from the blockchain database.
+    - SQL store APIs and Document store APIs will be exposed at post-PoC stage
+- All API access from the client to the blockchain DB is synchronous as most existing databases support synchronous APIs. Further, it is hard to write an application using asynchronous database APIs. Having said that, we can also support asynchronous APIs but not considered in this doc. 
```

Comment:
Almost all content is stale. It would be better to write it from scratch.

Reply:
I can write a readme from scratch with the latest information and new section/formatting.

Reply:
I mostly created this PR to have a placeholder and something to start with. We can put fixes in here to update with recent status.

Reply:
Agreed with @C0rWin

Reply:
I see. Okay then. I am merging this but I don't think design should be part of the readme. It should explain about the project briefly, a high level architecture, how to build and run the service, etc... 

We need to put the design,  transaction flow, tutorial for writing application should be part of the documentation.